package.json

```diff
@@ -16,7 +16,7 @@
     "@patternfly/quickstarts": "^5.1.0",
     "@patternfly/react-component-groups": "^5.5.4",
     "@patternfly/react-core": "^5.1.1",
-    "@patternfly/react-data-view": "^5.2.0",
+    "@patternfly/react-data-view": "^5.7.0",
```

Comment:
I noticed the toolbar has a wrong order of filters and actions, can you please update to v5.7.1 which fixes the issue? Thank you